addons/hr_timesheet/models/project_task.py

```diff
@@ -55,6 +59,17 @@ def _uom_in_days(self):
     def _compute_encode_uom_in_days(self):
         self.encode_uom_in_days = self._uom_in_days()
 
+    @api.depends('project_id.allow_timesheets')
+    def _compute_allow_timesheets(self):
+        for task in self:
+            task.allow_timesheets = task.project_id.allow_timesheets
+
+    def _search_allow_timesheets(self, operator, value):
+        query = self.env['project.project'].sudo()._search([
+            ('allow_timesheets', operator, value),
+        ])
+        return [('project_id', 'in', query)]
```

Comment:
I think we just have to do this then:
```suggestion
        return [('project_id', 'in', list(query))]
```

by doing that, the query will be executed and will return the ids fetched. 🙂 

(We will only do that in that version)

Reply:
Oh nice, that does seem to work